iohub/cli/cli.py

```diff
@@ -117,3 +118,18 @@ def convert(
         chunks=chunks,
     )
     converter.run(check_image=check_image)
+
+
+@cli.command()
+@click.help_option("-h", "--help")
+@click.argument(
+    "files",
+    nargs=-1,
+    required=True,
+    type=_DATASET_PATH,
+)
```

Comment:
`iohub` convert uses the `-i` convention, but you're dropping it here. Are you trying to match `iohub info`?

I think it makes sense to match the convention, especially because we want the wildcard behavior from the `-i`.